metadata-ingestion/tests/integration/kafka-connect/docker-compose.override.yml

```diff
@@ -38,6 +38,8 @@ services:
         #
         confluent-hub install --no-prompt confluentinc/kafka-connect-s3:10.5.1
         #
+        confluent-hub install --no-prompt snowflakeinc/snowflake-kafka-connector:2.0.0
```

Comment:
We don't need this right ? Anyway snowflake sink connector will have to be tested against mocked kafka connect responses.

Reply:
Just in case in future any developer again worked on this kafka connect snowflake sink connector and use this yml file to deploy kafka connect as I did. 
Still if you want me to remove let me know as we don't need this?

Reply:
can you comment it and revert the increase in timing ?. We would not want to increase our integration test timings.